astroid/brain/brain_namedtuple_enum.py

```diff
@@ -89,6 +90,10 @@ def infer_first(node):
     # set base class=tuple
     class_node.bases.append(base_type)
     # XXX add __init__(*attributes) method
+
+    if not enum and node.keywords and next((kw for kw in node.keywords if kw.arg == 'rename'), False):
+        _rename_namedtuple_attributes(attributes)
+
     for attr in attributes:
```

Comment:
First issue: currently instance attributes are added explicitly. All of tests refers to these `instance_attrs`, while they should be inferred via property decorators.